Kickstarter-iOS/Features/DashboardDeprecationBanner/DashboardDeprecationView.swift

```diff
@@ -0,0 +1,50 @@
+import Library
+import SwiftUI
+
+struct DashboardDeprecationView: View {
+  private let contentPadding = 12.0
+  private let imageSizeMultiplier = 1.5
+  private let deprecationDateText = formatted(dateString: "2023-08-14")
+
+  var body: some View {
+    HStack {
+      if let iconImage = image(named: "fix-icon", inBundle: Bundle.framework) {
```

Comment:
Always use conventions currently within the app.
In this case we can't be sure that "fix-icon" will be guaranteed to exist, so its' better to check for its' existence before creating the `Image`.